src/cron/generateInvites.ts

```diff
@@ -0,0 +1,55 @@
+import fastq from 'fastq';
+import { Cron } from './cron';
+import { Feature, FeatureValue, Invite } from '../entity';
+import { generateUUID } from '../ids';
+
+const cron: Cron = {
+  name: 'generate-invites',
+  handler: async (con, logger) => {
+    const featuresQuery = con
+      .createQueryBuilder()
+      .from(Feature, 'f')
```

Comment:
I don't think you need the alias actually.

Reply:
Alternative can't we handle it on pubsub or direct?
I can't recall why we choose the cron tbh.

Reply:
we can delegate this to a topic, similar to how the digest cron is simply publishing events that are then handled async. We would still be using cron, but we would be delegating processing to a worker. Is this what you are referring to?

Reply:
Closing as discussed on slack :)